packages/@okta/vuepress-site/docs/guides/deploy-to-production/overview/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Overview
+---
+You can deploy your Okta-protected app to many different hosting providers. You can also host it yourself.
+
+If you're developing a Single-Page Application (SPA), you'll need to host it on a web server and configure all paths to redirect to `index.html`. This is because your framework will handle the routing. You'll also need to force HTTPS.
```

Comment:
avoid future tense:

If you're developing a Single-Page Application (SPA), you need to host it on a web server...

This is because your framework handles the routing. You also need to force HTTPS.